README.md

```diff
@@ -1350,6 +1350,7 @@ If you're using [Antigen](https://github.com/zsh-users/antigen), you can test th
 * [leafia](https://github.com/Ghostrick/leafia-prompt) - Leafy prezto theme that shows `git` status information.
 * [leverage](https://github.com/gschnall/leverage) - Based on [minimal](https://github.com/robbyrussell/oh-my-zsh/blob/master/themes/minimal.zsh-theme), uses colors, and an extra `¬` character, to better distinguish the command line prompt from your output.
 * [lewis](https://github.com/lewisflude/oh-my-lewis) - Black, white and red theme. Shows `git` status information.
+* [lighthaus](https://github.com/lighthaus-theme/zsh) - A prompt that compliments the [Lighthaus](https://github.com/lighthaus-theme/lighthaus) theme. Shows _git_ information, github/gitlab logo and shows changes as and when they occur.
```

Comment:
Please change `_git_` to `git` for consistency with the rest of the list entries. Otherwise this looks good, thanks!

Reply:
> Please change `_git_` to `git` for consistency with the rest of the list entries. Otherwise this looks good, thanks!

Done. Thanks for the quick review.